packages/client-core/i18n/en/editor.json

```diff
@@ -72,7 +72,8 @@
     "unknownStatus": "Unknown Status",
     "CORS": "Possibly a CORS error",
     "urlFetchError": "Failed to fetch \"{{url}}\"",
-    "invalidSceneName": "Invalid input: Only alphanumeric characters, hyphens, and underscores are allowed."
+    "invalidSceneName": "Invalid input: Only alphanumeric characters, hyphens, and underscores are allowed.",
+    "invalidSceneNameLength": "Invalid input: Name must be between 4 and 64 characters"
```

Comment:
there is no need of a new message. Instead, we can rewrite the first message as: 
> Scene name must be 4-64 characters long, using only alphanumeric characters, hyphens, and underscores.